src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpointIT.java

```diff
@@ -0,0 +1,157 @@
+package uk.gov.ons.ctp.response.casesvc.endpoint;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.net.URL;
+import java.util.UUID;
+import java.util.concurrent.BlockingQueue;
+import javax.xml.XMLConstants;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Marshaller;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.embedded.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.rules.SpringClassRule;
+import org.springframework.test.context.junit4.rules.SpringMethodRule;
+import uk.gov.ons.ctp.response.casesvc.config.AppConfig;
+import uk.gov.ons.ctp.response.casesvc.message.notification.CaseNotification;
+import uk.gov.ons.ctp.response.casesvc.message.sampleunitnotification.SampleUnitParent;
+import uk.gov.ons.tools.rabbit.Rabbitmq;
+import uk.gov.ons.tools.rabbit.SimpleMessageBase;
+import uk.gov.ons.tools.rabbit.SimpleMessageListener;
+import uk.gov.ons.tools.rabbit.SimpleMessageSender;
+
+@Slf4j
+@ContextConfiguration
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class CaseEndpointIT {
+
+  @ClassRule public static final SpringClassRule SPRING_CLASS_RULE = new SpringClassRule();
+  @Rule public final SpringMethodRule springMethodRule = new SpringMethodRule();
+  @Rule public WireMockRule wireMockRule = new WireMockRule(options().port(18002));
+  @Autowired private ResourceLoader resourceLoader;
+  @LocalServerPort private int port;
+  @Autowired private ObjectMapper mapper;
+  @Autowired private AppConfig appConfig;
+
+  //  @Rule
+  //  public WireMockRule wireMockRule = new WireMockRule(options().port(18002));
+
+  @Test
+  public void ensureSampleAttributesReceived() throws Exception {
```

Comment:
Is the name of this method correct? Seems like it's more to do with the sampleUnitId should be on the case notification